src/sentry/models/useroption.py

```diff
@@ -38,26 +54,42 @@ def __setstate__(self, state):
         self.__dict__.update(state)
         self.__metadata = {}
 
-    def get_value(self, user, project, key, default=None):
-        result = self.get_all_values(user, project)
+    def get_value(self, user, key, default=None, **kwargs):
+        project = kwargs.get('project')
+        organization = kwargs.get('organization')
+
+        if organization and project:
+            raise NotImplementedError('this is not a supported use case, scope to project OR organization')
+        if organization:
+            result = self.get_all_values(user, None, organization)
```

Comment:
do you mean the error? it would work fine now, but I think that if someone tries to use org and project in the future, I don't want to silently ignore the project part